android/src/main/java/com/rnfs/RNFSManager.java

```diff
@@ -332,6 +333,7 @@ private WritableMap makeErrorPayload(Exception ex) {
     final Map<String, Object> constants = new HashMap<>();
     constants.put(NSDocumentDirectory, 0);
     constants.put(NSDocumentDirectoryPath, this.getReactApplicationContext().getFilesDir().getAbsolutePath());
+    constants.put(NSTemporaryDirectoryPath, null)
```

Comment:
@grabbou @johanneslumpe you are missing a `;` and this broke RNFS... @grabbou I will hunt you down and find you, and I will force you to eat Polish cabbage for 25 years for this riff-raff